recipe/bld.bat

```diff
@@ -1,3 +1,9 @@
+@echo on
+
+set "NM=llvm-nm.exe"
+set "LD=lld-link.exe"
+set "AR=llvm-ar.exe"
+set "RANLIB=llvm-ranlib.exe"
```

Comment:
This should be part of the activation, I just tried to add it manually to be sure...